CMake/CMakeTestISPCCompiler.cmake

```diff
@@ -1,3 +1,4 @@
+include(CMakeTestCompilerCommon)
```

Comment:
could you confirm is this works with Cmake 3.8/3.9?

Reply:
of course not :'(

Reply:
Does this fail with CMake v3.8? (If so, it shouldn't have merged?)

According to below, we say CMake 3.7 compatible: https://github.com/BlueBrain/CoreNeuron/blob/c03400580f6eeefdd4de994ade8aba609ac32042/CMakeLists.txt#L7 

We should implement this in CoreNEURON CI : https://github.com/neuronsimulator/nrn/pull/812

Reply:
no it doesn't fail

Reply:
what was failing was to use the new form of functions

Reply:
ok, thanks for clarification!